go/proto/deploy/builderd/v1/builder.proto

```
@@ -1,10 +1,10 @@
 syntax = "proto3";
 
-package builder.v1;
+package deploy.builderd.v1;
```

Buf (1.54.0)

Files with package "deploy.builderd.v1" must be within a directory "deploy/builderd/v1" relative to root but were in directory "go/proto/deploy/builderd/v1".

(PACKAGE_DIRECTORY_MATCH)